migrations/versions/0364_drop_old_column.py

```diff
@@ -0,0 +1,35 @@
+"""
+
+Revision ID: 0364_drop_old_column
+Revises: 0363_cancelled_by_api_key
+Create Date: 2022-01-25 18:05:27.750234
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0364_drop_old_column'
+down_revision = '0363_cancelled_by_api_key'
+
+
+def upgrade():
+    op.get_bind()
+    op.execute("UPDATE broadcast_message SET created_by_api_key_id=api_key_id")  # move data over
+    op.create_check_constraint(
+        "ck_broadcast_message_created_by_not_null",
+        "broadcast_message",
+        "created_by_id is not null or created_by_api_key_id is not null"
+    )
+    op.drop_column('broadcast_message', 'api_key_id')
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('broadcast_message', sa.Column('api_key_id', postgresql.UUID(), autoincrement=False, nullable=True))
+    op.get_bind()
```

Comment:
Do we need this? I can see it in other migrations, but I'm not sure what it's for.

Reply:
https://alembic.sqlalchemy.org/en/latest/ops.html?#:~:text=For%20full%20interaction%20with%20a%20connected%20database%20where%20parameters%20can%20also%20be%20used%20normally%2C%20use%20the%20%E2%80%9Cbind%E2%80%9D%20available%20from%20the%20context%3A

looks like it;s not needed in this case, as that fn just returns something.

some alembic scripts do use the return value tho for passing through to other functions or running `.execute` on though.

Reply:
Ahh, good spot, I just copied from earlier migrations, I will get rid of it ☺️